Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM ubuntu:16.04
 MAINTAINER Adrien Leravat <Pixep@users.noreply.github.com>
 
 # Install prerequisites
-RUN echo nameserver 8.8.8.8 > /etc/resolv.conf
+CMD echo nameserver 8.8.8.8 >> /etc/resolv.conf
```

Comment:
Are you sure about using `CMD` here? There can only be one CMD, and it is executed when lunched, vs built. I suspect this is actually ignored entirely.

Reply:
Yup, you are correct it will be skipped, I resolved my issue by changing the `FROM ...`.
Thanks